syntax = "proto3";

package api;

message PrepayRequest {
  int32 pay_fee = 1;
  string payer_id = 2;
  string pay_method = 3;
  string type = 4;
  // 商品ID，单个商品时有效
  string commodity_id = 5;
}

message QueryRequest {
  string transaction_id = 1;
  string pay_method = 2;
}

message PrepayResponse {
  string transaction_id = 1;
  // 支付方式
  string pay_method = 2;
  // 当面付功能
  string qrcode_url = 3;
}

message QueryResponse {
  string transaction_id = 1;
  bool success = 2;
  string msg = 3;
  string sub_msg = 4;
}
